.github/workflows/daily.yml

```
@@ -0,0 +1,10 @@
+name: Daily
+on:
+  schedule:
+    - cron: "0 1 * * *"
```

[optional] to help developers who aren't familiar with cron

```suggestion
  schedule:
    - cron: "0 1 * * *" # every day at 1 AM
```